src/modules/numberplate/codes.skip.ts

```diff
@@ -0,0 +1,290 @@
+export interface CarDataset {
+	[index: number]: string;
+}
+export interface CategoryDataset {
+	[index: string]: string;
+}
+export type MotorcycleDataset = CarDataset;
+
+export interface PlateDataset {
+    Car: CarDataset,
+    Motorcycle: MotorcycleDataset,
+    Category: CategoryDataset
+}
+
+export const plateDataset: PlateDataset = {
+	Car: {
+		10: "تهران",
```

Comment:
I think this file is not prettified by prettier, isn't that so?

Reply:
I ran both prettier & prettier:ci  and got no errors